internal/types/secret_test.go

```diff
@@ -65,6 +65,9 @@ func TestRoundTripRedactExternalServiceConfig(t *testing.T) {
 			Dependencies: []string{"placeholder"},
 		},
 	}
+	pagureConfig := schema.PagureConnection{
+		Url: "https://src.fedoraproject.org",
+	}
```

Comment:
```suggestion
	pagureConfig := schema.PagureConnection{
		Url: "https://src.fedoraproject.org",
		Token: "top secret token",
	}
```

The current code will test the `""` string code path, whereas I think what you want to test here is that the token is redacted correctly if it is present.